deployment/roles/toy.database/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Install PostgreSQL
+  apt: pkg="{{ item }}" state=present
+  with_items:
+    - postgresql
+    - libpq-dev
+    - postgresql-client
+    - postgresql-client-common
+
+- name: Add UbuntuGIS apt repository
+  apt_repository:
+    repo: ppa:ubuntugis/ppa
+
+- name: Update APT cache
+  apt: update_cache=yes
```

Comment:
You can just include this in the task below, I think that's how we normally do it.